scripts/setup-8-validators.sh

```diff
@@ -1,13 +1,12 @@
-#!/bin/sh
+#!/bin/bash
 
-PRIVATE_KEY_PATH=PUTPRIVKEYPATHHERE
+PRIVATE_KEY_PATH=~/priv
```

Comment:
can this be a flag?

Reply:
It is, I just have a default value there.